### PR TITLE
make unique dask work

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by martindurant** [comment]
Fixes #170 

Could use a test.

@mrocklin , thoughts no the conversation in the linked issue?

**Comment by CJ-Wright** [comment]
Ping this issue.

File: streamz/tests/test_dask.py [hunk 1]
```diff
@@ -178,3 +178,14 @@ def add(x, y, z=0):
...
+
+@gen_cluster(client=True)
+def test_unique(c, s, a, b):
+    source = Stream(asynchronous=True)
+    L = source.scatter().unique(key=lambda x: x.key).gather().sink_to_list()
+
+    for i in range(5):
+        yield source.emit(1)
+
+    assert L == [1]
```
[the remark this answers is not in the document]
Comment (CJ-Wright) [reply streamz/tests/test_dask.py]: Is that dependent on what the user wants? If they wanted to inspect the actual value they could request it via the key syntax, right?
Is there a better way to run the inspection?
I guess unique is a special case of filter but then we are back at the filter no-op problem again.

**Comment by mrocklin** [comment]
If I wanted to build unique I would probably use accumulate.  I don't have
a burning desire to do this though, so I suggest that we just leave it to
users for now.

**Pull request closed without merging**